src/Nancy/Responses/DefaultXmlSerializer.cs

```diff
@@ -64,19 +77,19 @@ public void Serialize<TModel>(MediaRange mediaRange, TModel model, Stream output
             }
         }
 
-        private static bool IsXmlType(string contentType)
+        private static bool IsXmlType(MediaRange mediaRange)

```

Comment:
The signature was updates since stuff now speaks `MediaRange`, but yeah that method is probably better of accepting a string so we only `ToString` once. Will revert